source/userguide/auto_annotator.rst

```diff
@@ -304,23 +382,115 @@ For example, we can restrict the output of the previous example by doing the fol
 	
 	allowed_entites = ["sys_org", "sys_amount-of-money", "sys_time"]
 	sentence = "Apple stock went up $10 last monday."
-	sa.parse(sentence=sentence, entity_types=allowed_entities) 
+	mla.parse(sentence=sentence, entity_types=allowed_entities)
+
+Working with Non-English Sentences
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The :class:`MultiLingualAnnotator` will use the language and locale specified in the :attr:`LANGUAGE_CONFIG` (:attr:`config.py`) if it used through the command-line.
+
+.. code-block:: python
+	
+	LANGUAGE_CONFIG = {'language': 'es'}
+
+Many Spacy non-English NER models have limited entity support. To overcome this, the :class:`MultiLingualAnnotator` translates the sentence to English and detects entities
+using the English NER model. The English detected entities are compared against duckling candidates for the non-English sentence. Duckling candidates with a match between the type and value of the entity or the translated body text
+are selected. If a translation service is not available, the :class:`MultiLingualAnnotator` selects the duckling candidates with the largest non-overlapping spans. The sections below describe the steps to setup the annotator depending on whether a translation service is being used.
```

Comment:
Seems to convey we only return the entities that overlap with the english NER and non-english duckling models. Consider adding another sentence in the end to clarify that the final list includes these overlapping entities + non-English spacy entites?

```
Duckling candidates with a match between the type and value of the entity or the translated body text
are selected and returned as valid entities in addition to the ones detected by the non-English NER models. 
```

Reply:
Many Spacy non-English NER models have limited entity support. To overcome this, the :class:`MultiLingualAnnotator` translates the sentence to English and detects entities
using the English NER model.
```suggestion
Many Spacy non-English NER models have limited entity support. To overcome this, in addition to the entities detected by non-English NER models, the :class:`MultiLingualAnnotator` translates the sentence ...
```

Reply:
Done